src/Microsoft.DotNet.Interactive.HttpRequest/HttpRequestKernel.cs

```diff
@@ -56,31 +54,10 @@ public HttpRequestKernel(string? name = null, HttpClient? client = null)
         KernelInfo.DisplayName = $"{KernelInfo.LocalName} - HTTP Request";
 
         _client = client ?? new HttpClient();
-        var setHost = new Command("#!set-host", "Sets the host name to be used when sending requests using relative URLs");
-        setHost.AddArgument(_hostArgument);
-        setHost.SetHandler(context =>
-        {
-            var host = context.ParseResult.GetValueForArgument(_hostArgument);
-            BaseAddress = host;
-        });
-        AddDirective(setHost);
-
+    
         RegisterForDisposal(_client);
     }
-
-    public Uri? BaseAddress

```

Comment:
Looks like the API compatibility test baseline needs to be updated for removal of this public API.

Reply:
There is also a public `SetValue` function below. Does it need to be part of the public API considering the same can be accomplished via a `SendValue` commands?

Reply:
That can be removed I suspect.

Reply:
if the gesture is like value sharing then yes, we should remove that.